### PR TITLE
rsync: update to 3.2.2

### PR DESCRIPTION
**Pull request by AndWac**
I reworked the file patch but I don't know whether it's really needed (I don't understand its description).
Furthermore there's a warning (already in our actual version):

configure.sh: WARNING: netinet/ip.h: present but cannot be compiled
configure.sh: WARNING: netinet/ip.h:     check for missing prerequisite headers?
configure.sh: WARNING: netinet/ip.h: see the Autoconf documentation
configure.sh: WARNING: netinet/ip.h:     section "Present But Cannot Be Compiled"
configure.sh: WARNING: netinet/ip.h: proceeding with the compiler's result
configure.sh: WARNING:     ## --------------------------------------------------- ##
configure.sh: WARNING:     ## Report this to http://rsync.samba.org/bugzilla.html ##
configure.sh: WARNING:     ## --------------------------------------------------- ##

### CODE REVIEW

**Comment by pyhalov** [comment]
The patch was needed  to avoid embedding path to build area in compiled binary and in particular in error message.

File: components/network/rsync/Makefile [hunk 1]
```diff
@@ -22,43 +22,41 @@
...
 
 COMPONENT_TEST_TRANSFORMER = $(NAWK)
 COMPONENT_TEST_TRANSFORMS = "'/^PASS|^SKIP|^FAIL|^XFAIL/'"
 
-# common targets
-build:		$(BUILD_64)
-
-install:	$(INSTALL_64)
-
-test:		$(TEST_64)
```
Comment (pyhalov) [inline components/network/rsync/Makefile:61]: Is fakeroot needed for tests? If it is, add it as build dependency.
Otherwise looks good.

Comment (AndWac) [reply components/network/rsync/Makefile:61]: How to use fakeroot? I haven't found any reference to it in the repository.

Comment (pyhalov) [reply components/network/rsync/Makefile:61]: It's used by rsync testsuite if found. Just add it to REQUIRED_PACKAGES, as test results change if it's not present on the build system.

**Review by pyhalov — APPROVED** [review]

**Pull request merged**